src/queries/fetcher.ts

```diff
@@ -0,0 +1,33 @@
+function checkStatus(response: Response) {
+  if (response.status >= 200 && response.status < 300) {
+    return Promise.resolve(response);
+  }
+  return Promise.reject(new Error(response.statusText));
+}
+
+function getResponseJson(response: Response) {
+  return response.json();
+}
+
+export function get(url: string) {
+  return fetch(url, {
```

Comment:
Please, rewrite to async/await try/catch.

Reply:
not ready yet

Reply:
done